python/gsw/main.py

```diff
@@ -93,6 +99,9 @@ def check_for_email(self):
 
                     #handles uplink confirmations
                     if email_subject.find("SBD Mobile Terminated Message Queued for Unit: ")==0:
```

Comment:
It would be nice if these two email processings could be unit-tested. We can defer this though